infra/src/main/resources/reference.conf

```diff
@@ -18,4 +18,10 @@ scaladex {
     # follow this tutorial to create your token https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token
     token = ${?GITHUB_TOKEN}
   }
+  filesystem {
```

Comment:
so the goal is to remove dataPath and replace it by fylesystemConfig ?

Reply:
Yes I would like to remove `DataPaths` completely and move all methods that read/write into the filesystem to the `LocalStorageRepo` class. So that we have a better view of all that depends on reading/writing to disk. Also to create a mock of `LocalStorage` and test the things that depends on it.

That's quite some work though.

So in the end `LocalStorageRepo` should be the only thing consuming the `FilesystemConfig`.

Reply:
which make sense!